packages/superset-ui-legacy-plugin-chart-table/src/index.ts

```diff
@@ -16,6 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+/// <reference path="../types/external.d.ts" />
```

Comment:
Is this necessary?

Reply:
This is necessary when importing from `superset-frontend` directly. Figured it's best to leave it there since it doesn't break anything anyway.

Reply:
Removed to fix linting error. :D